docs/api/system-preferences.md

```diff
@@ -380,6 +380,30 @@ You can use the `setAppLevelAppearance` API to set this value.
 Sets the appearance setting for your application, this should override the
 system default and override the value of `getEffectiveAppearance`.
 
+### `systemPreferences.isTouchIDAvailable()` _macOS_
+
+Returns `Boolean` - whether or not this device has the ability to use Touch ID.
+
+**NOTE:** This API is only available on macOS Sierra 10.12.2 or newer.
```

Comment:
_(pedantic)_ This API appears to be available on all versions of macOS; it just returns `false` on older versions. The current phrasing implies that app devs need to test their OS version before calling this function.

How about:

> **NOTE:** Touch ID is only available on macOS Sierra 10.12.2 or newer.

Reply:
The problem with this is that `TouchID` is available on versions prior to `10.12.2`. The API to interact with it, however, is not, which causes this issue.